### PR TITLE
TS migration of organization & me service

### PR DESCRIPTION
**Pull request by future-pirate-king**

### CODE REVIEW

File: app/services/me.ts [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  }
+
+  async getMembership() {
+    const org = await this.queryOrganizationMe;
+    const user_id = org.id;
+
+    return this.store.findRecord('organization-member', user_id);
+  }
+
+  // get partner() {
```
Comment (Yibaebi) [inline app/services/me.ts:38]: Commented code?

**Review by Yibaebi — APPROVED** [review]

File: app/services/me.ts (hunk 1 above)
Comment (future-pirate-king) [reply app/services/me.ts:38]: not sure the purpose of this getter as `organization-me` model don't have `partner` property, so just commented for future reference

Comment (Praseetha-KR) [reply app/services/me.ts:38]: We can delete this @future-pirate-king , I don't think `partner` is used anywhere

Comment (future-pirate-king) [reply app/services/me.ts:38]: yeah will do that

**Review by Praseetha-KR — APPROVED** [review]

**Pull request merged**